app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\Request;
+
+/**
+ * @param $id
+ * @param $posts
+ * @return mixed|void
+ */
+function find($id, $posts)
```

Comment:
It is not good practice to add standalone functions. It kills maintainability and is poor OO-design. Add methods instead, just like in your game project.